src/routes/artist.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+
+const artistController = require("../controllers/artist");
+
+const artistRouter = express.Router();
+
+artistRouter.post("/artists", artistController.createArtist);
+artistRouter.get("/artists", artistController.readArtist);
+artistRouter.get("/artists/:id", artistController.getArtistbyId);
+artistRouter.put("/artists/:id", artistController.updateArtist);
+artistRouter.patch("/artists/:id", artistController.updateArtist);
+artistRouter.delete("/artists/:id", artistController.deleteArtist);
```

Comment:
Following from my comment in app.js these routes shouldn't have `"/artists"` in them as that should be added in app.js, i.e.
```
artistRouter.post("/", artistController.createArtist);
artistRouter.get("/", artistController.readArtist);
artistRouter.get("/:id", artistController.getArtistbyId);
artistRouter.put("/:id", artistController.updateArtist);
artistRouter.patch("/:id", artistController.updateArtist);
artistRouter.delete("/:id", artistController.deleteArtist);
```